CryptoLightNews/MainTabBarController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  MainTabBarController.swift
+//  CryptoLightNews
+//
+//  Created by Richard Price on 15/12/2022.
+//
+
+import UIKit
+
+class MainTabBarController: UITabBarController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        view.backgroundColor = .darkGray
+    }
+    
+    private func setupNavControllers(with rootViewController: UIViewController, title: String, image: UIImage) -> UINavigationController {
+        let navController = UINavigationController(rootViewController: rootViewController)
+        rootViewController.title = title
+        navController.tabBarItem.title = title
+        navController.tabBarItem.image = image
+        return navController
+    }
+    
+    private func setupUI() {
+        viewControllers = [
+            setupNavControllers(with: UIViewController(), title: "News", image: #imageLiteral(resourceName: "cryptoNews")),
+            setupNavControllers(with: UIViewController(), title: "Prices", image: #imageLiteral(resourceName: "prices_new")),
+        ]
+        //modify tab bar insets
+                guard let items = tabBar.items else {return}
+                for item in items {
+                    item.title = ""
+                    item.imageInsets = UIEdgeInsets(top: 2, left: 0, bottom: -2, right: 0)
+                }
+        //setup tab colorScheme
```

Comment:
So if you need a comment here, maybe you need a method? 😉

Reply:
moved into function called updateTabColorScheme